Sources/Plugins/OAPlugin.mm

```diff
@@ -385,7 +385,7 @@ - (void)prepareExtraTopPoiFilters:(NSSet<OAPOIUIFilter *> *)poiUIFilters
 
 + (void)onPrepareExtraTopPoiFilters:(NSSet<OAPOIUIFilter *> *)poiUIFilters
 {
-    for (OAPlugin *plugin in [self getEnabledPlugins])
+    for (OAPlugin *plugin in [self.class getEnabledPlugins])
```

Comment:
No need for this, it is correct to call self when calling another class static methods